src/inmanta/env.py

```diff
@@ -69,17 +124,106 @@ def install_from_source(cls, paths: List[LocalPackagePath]) -> None:
             raise Exception("install_from_source requires at least one package to install")
         # make sure we only try to install from a local source
         explicit_paths: Iterator[LocalPackagePath] = (
-            LocalPackagePath(path=os.path.join(".", path.path), editable=path.editable) for path in paths
+            LocalPackagePath(path=os.path.join(".", path.path, ""), editable=path.editable) for path in paths
```

Comment:
why `,""`

Reply:
To make sure a `/` is added at the end. Otherwise pip doesn't look for a local path iirc.

Reply:
I would add that as a comment, because it is important.

Reply:
done